plothist/hep_plotters.py

```diff
@@ -121,6 +121,29 @@ def compare_data_mc(
             uncertainties_high**2,
         )
         data_hist[:] = np.stack([data_hist.values(), data_variances], axis=-1)
+    elif comparison_kwargs["comparison"] == "ratio":
+        if comparison_kwargs["ratio_uncertainty"] == "split":
+            np.seterr(divide="ignore", invalid="ignore")
+            # Compute asymmetrical uncertainties to plot_comparison()
+            comparison_kwargs["yerr"] = [
```

Comment:
if user provides an "yerr", this will overwrite it. This comportment might be ok, but needs to be reported in docstring.

Reply:
fixed using `set_default()`